packages/rocketchat-videobridge/client/public/external_api.js

```diff
@@ -0,0 +1,377 @@
+     * @param arguments array of arguments
+     */
+    JitsiMeetExternalAPI.prototype.executeCommand = function(name,
+                                                             argumentsList) {
+        var argumentsArray = argumentsList;
+        if (!argumentsArray)
+            argumentsArray = [];
+        var object = {type: "command", action: "execute"};
+        object[name] = argumentsArray;
+        this.sendMessage(object);
+    };
+
+    /**
+     * Executes commands. The available commands are:
+     * displayName - sets the display name of the local participant to the value
+     * passed in the arguments array.
+     * toggleAudio - mutes / unmutes audio with no arguments
+     * toggleVideo - mutes / unmutes video with no arguments
+     * filmStrip - hides / shows the film strip with no arguments
+     * @param object the object with commands to be executed. The keys of the
+     * object are the commands that will be executed and the values are the
+     * arguments for the command.
+     */
+    JitsiMeetExternalAPI.prototype.executeCommands = function (object) {
+        object.type = "command";
+        object.action = "execute";
+        this.sendMessage(object);
+    };
+
+    /**
+     * Adds event listeners to Meet Jitsi. The object key should be the name of
+     * the event and value - the listener.
+     * Currently we support the following
+     * events:
+     * incomingMessage - receives event notifications about incoming
+     * messages. The listener will receive object with the following structure:
+     * {{
+     *  "from": from,//JID of the user that sent the message
+     *  "nick": nick,//the nickname of the user that sent the message
+     *  "message": txt//the text of the message
+     * }}
+     * outgoingMessage - receives event notifications about outgoing
+     * messages. The listener will receive object with the following structure:
+     * {{
+     *  "message": txt//the text of the message
+     * }}
+     * displayNameChanged - receives event notifications about display name
+     * change. The listener will receive object with the following structure:
+     * {{
+     * jid: jid,//the JID of the participant that changed his display name
+     * displayname: displayName //the new display name
+     * }}
+     * participantJoined - receives event notifications about new participant.
+     * The listener will receive object with the following structure:
+     * {{
+     * jid: jid //the jid of the participant
+     * }}
+     * participantLeft - receives event notifications about the participant that
+     * left the room.
+     * The listener will receive object with the following structure:
+     * {{
+     * jid: jid //the jid of the participant
+     * }}
+     * @param object
+     */
+    JitsiMeetExternalAPI.prototype.addEventListeners
+        = function (object) {
+
+        var message = {type: "event", action: "add", events: []};
+        for(var i in object)
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Keyword "for" must be followed by whitespace.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2748851505/issues/source?bid=3199634&fileBranchId=3342079#l196)